docker/Dockerfile_template

```diff
@@ -20,10 +20,8 @@ ENV ROUTE53_ACCESS_KEY <aws_route53_access_key>
 ENV ROUTE53_SECRET_ACCESS_KEY <aws_route53_secret_access_key>
 ENV OCM_OFFLINE_TOKEN <ocm_offline_token>
 ENV OBSERVATORIUM_CONFIG_ACCESS_TOKEN <observatorium_config_access_token>
-ENV MAS_SSO_CLIENT_ID <mas_sso_client_id>
-ENV MAS_SSO_CLIENT_SECRET <mas_sso_client_secret>
-ENV OSD_IDP_MAS_SSO_CLIENT_ID <osd_idp_mas_sso_client_id>
```

Comment:
What is then used for IDP SSO?

Reply:
mas-sso is still used for IDP.
But since the midstream test are running against rh_sso, we cannot configure the IDP with it hence this removal

Reply:
That means there's no way to access the cluster using our users in case we need it right?

Reply:
> That means there's no way to access the cluster using our users in case we need it right?

I think this has always been the case even with the current setup, we do not have all the right access to the clusters since we are not part of the default user list: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/35cf0417534cc5bb5b390f6d97586cb66f811549/config/kafka-sre-user-list.yaml and https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/35cf0417534cc5bb5b390f6d97586cb66f811549/config/read-only-user-list.yaml

And I've been always using kubeconfig to access the nightly cluster. 
With the change in this PR https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/35cf0417534cc5bb5b390f6d97586cb66f811549/internal/kafka/test/helper.go#L72, IDP configuration will be disabled completely and kubeconfig access option will be the default.